### DISCUSSION TITLE
kwargs["winmode"] = 0   not correct, or?

### DISCUSSION THREAD

**Discussion by rolfmertig** [rjfarmer/gfort2py#58] [Q&A]
Only if I change the 0 to a 1 in this line:

https://github.com/rjfarmer/gfort2py/blob/a0aead035c1c10926f772fbb8ab8a59d748051c1/gfort2py/gfort2py.py#L54

I can get fFort to work on Windows, using a recent Python version.

So what is the rationale to have this limit to python version greater or equal than 3.8 ?

**Comment by rjfarmer** [comment]
winmode adjusts how the paths are searched for dll's https://docs.python.org/3/library/ctypes.html#ctypes.WinDLL It was only added in 3.8, hence the version check. Setting to 0 should make it the easiest to load dll's. 

Do you use a full path or relative path for the dll load?
What error messages do you get when it fails?
What python version are you using? and whats providing the gfortran compiler for you (cygwin, mingw, wsl, etc)?
